web-registry/src/components/organisms/BridgedEcocreditsTable/BridgedEcocreditsTable.utils.tsx

```diff
@@ -0,0 +1,39 @@
+import { BridgedTxStatus } from 'types/ledger/ecocredit';
+
+import { Link } from 'components/atoms';
+
+import {
+  BRIDGED_STATUSES,
+  STATUS_COMPLETE,
+  STATUS_ERROR,
+  STATUS_PENDING,
+} from './BridgedEcocreditsTable.constants';
+
+export const getNote = (
```

Comment:
Should this be a component?
While it is not critical here, when returning a component from a function it prevents React from keeping them in its render tree which can cause some blinking if there are too many renders.